demo/mmdetection_cfg/cascade_rcnn_x101_64x4d_fpn_coco.py

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
should configs be ruled out?

Reply:
Yes. Will fix it.

Reply:
Fixed by adding "exclude" to `pre-commit-config.yaml`